README.md

```diff
@@ -152,14 +152,23 @@ quarkus:
         url: ldap://override-when-needed
       identity-mapping:
         search-base-dn: ou=users,o=YourCompany,c=ES
+# Flyway to create PIM schema
+  flyway:
+    connect-retries: 10
+    table: flyway_pim_history
+    migrate-at-start: true
+    baseline-on-migrate: true
+    baseline-version: 1.0
+    baseline-description: PimDB
+    sql-migration-prefix: h2 (1)        
 # Quartz configuration
   quartz:
-    store-type: ram
+    store-type: jdbc-cmt
```

Comment:
Shouldn't it be `ram` or `jdbc-tx` according to the paragraph below?

Reply:
same